src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/AuthenticationController.kt

```diff
@@ -41,4 +41,14 @@ class AuthenticationController(context: ActionContext,
             is HTMLForm.InvalidForm -> FailedAuthentication(validationResult.problem)
         }
     }
+
+    fun setShinyCookie(): String
+    {
+        val user = userRepository.getUserByUsername(context.username!!)
+        val montaguUser = userRepository.getMontaguUserByEmail(user.email)
```

Comment:
Why do we fetch the user from the database twice?

Reply:
We don't have the email initially, and don't get the roles/permissions in the first call. I could add a new repo method, I was just being lazy and re-using existing methods.